### PR TITLE
mb_review

### PR DESCRIPTION
**Pull request by polinonicholas**

### CODE REVIEW

**Review by mikebad1984 — APPROVED** [review]
It’s usually recommended to have a commit with only format changes when you start using a formatter. Then make other changes in a follow up commit. Here’s your first approval! Before clicking merge make sure to look at how the diff in github looks. Like I said, even though I might do my own commits I use this workflow and approve myself because it’s a different way to look at things and catch a few small errors (like a variable that was misspelled)

File: compile/fanduel_single_game_template.py [hunk 1]
```diff
@@ -28,50 +26,65 @@
 # stack_info = stack_auto[['s_z', 'p_z', 'z', 'points', 'salary', 'stacks', 'sp_mu']].sort_values(by='points', ascending=False)
 
 
-# def build_lineups(self, 
-#                       lus = 150, 
-#                       index_track = 0, 
+# def build_lineups(self,
+#                       lus = 150,
```
Comment (mikebad1984) [inline compile/fanduel_single_game_template.py:30]: Making a comment on a random line so you can see how this works. Nothing wrong here, but you can see how the context helps around “oh found an issue here”

Comment (mikebad1984) [reply compile/fanduel_single_game_template.py:30]: You can comment here, and when a change has been agreed to, you can make it and then resolve the conversation when you’re done. A nice todo

Comment (polinonicholas) [reply compile/fanduel_single_game_template.py:30]: Thank you!

**Pull request merged**